pom.xml

```diff
@@ -145,6 +145,12 @@
       <artifactId>appengine-plugins-core</artifactId>
       <version>0.4.1</version>
     </dependency>
+    <dependency>
+      <!-- bundle: org.apache.commons.compress -->
```

Comment:
Do you think it's worth mentioning that this version comes from appengine-plugins-core?  We haven't done this for the other dependencies and I somewhat regret it (e.g., what restrictions do we have for updating jackson?).

Reply:
Good idea.